spec/wibox/widget/textbox_spec.lua

```diff
@@ -5,6 +5,17 @@
 
 local textbox = require("wibox.widget.textbox")
 
+local test_dpi_value = 192
+_G.screen = {
```

Comment:
This is a general note:

I think I have not seen usage of `_G` outside of spec. I think it's a good practice when accessing to global C API objects.

Are there particular reason for using `_G` only in specs?
Should we recommend to use `_G` as a general good practice to use everywhere in the future?

Reply:
i don't think here is appropriate time and place to discuss it